emails/utils.py

```diff
@@ -275,3 +277,13 @@ def decrypt_reply_metadata(key, jwe):
     e.deserialize(jwe)
     e.decrypt(k)
     return e.plaintext
+
+def get_message_content_from_s3(bucket, object_key):
+    emails_config = apps.get_app_config('emails')
+
+    # attachment = SpooledTemporaryFile()
+    with open('temp_file', 'w+b') as f:
```

Comment:
Can we try https://stackoverflow.com/a/38910315 so we don't have to touch the file-system? Putting any files-from-the-internet in our app server's file-system makes me nervous.

Reply:
Thanks for bringing this to my attention. I was using BufferedRandom to temporarily store the data and I agree with your concern and mediation on using streamed data instead. Instead of following the answer on StackOverflow, I used the [get_object](https://boto3.amazonaws.com/v1/documentation/api/1.9.42/reference/services/s3.html?highlight=delete_object#S3.Client.get_object) function in the S3 client which does the same thing and less code :)